### ISSUE TITLE
Support increased browser base font size

### ISSUE THREAD

**Issue by thecodercoder** [mdn/fred#967]
### Summary

The website content is too cramped in the columns if the browser base font size is increased, for example to 36 (as shown in screencast). I believe this is due to the media queries being set in pixels. This means that when the base font size is changed, all the text scales up with it, but the breakpoints still occur at the same pixel width, causing the cramped text.

I think a good solution would be to switch to em units for media queries, which will cause the breakpoints to scale up along with all the text, and be a better user experience for users who use the browser's base font size to increase text size. (See references below)

### URL

https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme

### Reproduction steps

1. Go to any detail page on MDN
2. Change browser base font size to something greater than 16
3. The bigger the font size, the more the issue of cramped columns of text will be emphasized

### Expected behavior

Ideally increasing the base font size would increase the text size, but keep the overall general layout of the website the same, simply scaled up.

### Actual behavior

When base font size is increased, for example to 32, the layout remains in the desktop version with 3 columns due to the media query breakpoint being in pixels. But the text scales up with the base font size change, resulting in too-large text in too-narrow columns.

### Device

Laptop

### Browser

Firefox

### Browser version

Stable

### Operating system

Windows

### Screenshot

https://user-images.githubusercontent.com/37986728/157986335-df384848-9b36-4394-8b6e-4c00b1f95089.mov

### Anything else?

There are some articles on the topic of media queries that seem to be referenced often, so I have included them here:

Zell Liew wrote in 2016 that em units should be used due to px and rem units causing browser issues in Safari if the html font-size is changed (e.g. "200%") or zoomed in, and for accessibility reasons. I confirmed that the bugs with px and rem units do still seem to happen with Safari.
https://zellwk.com/blog/media-query-units/

Adam Wathan wrote in 2017 that em units result in buggy behavior in Safari if zooming in. However I and others have confirmed that this bug in Safari no longer exists.
https://adamwathan.me/dont-use-em-for-media-queries/

Additionally, I have some screencasts of a demo showing the behavior on my Twitter:
https://twitter.com/thecodercoder/status/1502131251481829387

With this in mind, I suggest that media queries will operate best for accessibility if em units are used instead of pixels. Please let me know if there's any other information you might need! (Love the new redesign!)

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/mdn/yari/discussions).

**Comment by e3dio** [comment]
You are right, I tried changing the 3 column media query from 1200px to the equivalent 75rem, here are the before and after for Chrome's largest font-size (24px / 150%) at 1200px width: 

# Before (>1200px):

![image](https://user-images.githubusercontent.com/85405955/158005854-9f4fd2e0-41d2-42ae-bbe2-33d3f49e0c3a.png)

# After (>1200px):

![image](https://user-images.githubusercontent.com/85405955/158005872-f459a06e-207c-4798-9dbe-89df801aa139.png)

I think this fits with the intended design. Also should change max-width: 1440px to 90rem

**Comment by thecodercoder** [comment]
One quick note-- my proposal was to use "em" units, not "rem" units, for the media queries. There are some Safari bugs with using "rem" units while zooming the browser that you can read more about in Zell Liew's article, linked in my initial issue. But yes, your second set of screenshots are what I was thinking would be a more optimized design.

**Comment by e3dio** [comment]
6 years later and Safari still has the same bugs? Safari is always causing trouble, it's the new Internet Explorer :P

**Comment by caugner** [comment]
@thecodercoder Is this still an issue?

I tried [this MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme) both with Firefox 146 Nightly and Chrome 141, with an increased base font size of 32px: The page appears to be mostly readable, and most importantly it shows in two columns, not three.

I have noticed the following nits:

- The menu overflows the screen to the right.
- The Interactive Example in the [Try it](https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme#try_it) doesn't wrap, so the individual options have scrollbars.

If you could point out the most important pain points, that would be helpful for us. 🙏

**Comment by globau** [comment]
Putting into the backlog as the major issues have been addressed (as per https://github.com/mdn/fred/issues/967#issuecomment-3441834562)
